frontend/packages/console-app/src/components/lightspeed/Lightspeed.scss

```diff
@@ -0,0 +1,106 @@
+@import '~@patternfly/patternfly/sass-utilities/all';
+@import '../../../../../public/style/vars';
+
+:root {
+  --popover-border-radius: 12px;
+  --popover-button-size: var(--pf-v5-global--spacer--2xl);
+  --popover-margin: var(--pf-v5-global--spacer--lg);
+  --popover-bottom: calc(
+    var(--popover-button-size) + var(--popover-margin) + var(--pf-v5-global--spacer--sm)
+  );
+}
+
+.lightspeed__popover,
+.lightspeed__popover-button {
+  box-shadow: var(--pf-v5-global--BoxShadow--lg);
+  position: absolute;
+  right: var(--popover-margin);
+  z-index: 199;
+}
+
+.lightspeed__popover {
+  border-radius: 12px;
+  border: var(--pf-v5-global--BorderWidth--sm) solid var(--pf-v5-global--BorderColor--100);
+  bottom: var(--popover-bottom);
+  overflow: hidden;
+}
+
+.lightspeed__popover--collapsed {
+  top: calc(var(--popover-margin) * 5);
+  width: 90%;
+
+  @media (min-width: $pf-v5-global--breakpoint--sm) {
+    top: auto;
+    height: 70vh;
+  }
+
+  @media (min-width: $pf-v5-global--breakpoint--md) {
+    width: 50%;
+  }
+
+  @media (min-width: $pf-v5-global--breakpoint--lg) {
+    width: 40%;
+  }
+
+  @media (min-width: $pf-v5-global--breakpoint--xl) {
+    height: 55vh;
+  }
+}
+
+.lightspeed__popover--expanded {
+  height: calc(100% - var(--popover-bottom) - var(--popover-margin));
+  width: calc(100% - 2 * var(--popover-margin));
+}
+
+.lightspeed__popover-button {
```

Comment:
FYI @kyoto.  And note the element is a `<Button>` for accessibility.

Reply:
Thanks! Changed to a `<Button>` in the plugin by https://github.com/openshift/lightspeed-console/pull/317